src/Resources/DoctrineMigrations/pdo_pgsql/Version20210412122119.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
This reminds me that I wanted to talk about abandonning mysql and sqlite DoctrineMigrations as they are not systematically maintained.

Reply:
There are maintained. Good point I just forgot